services/web/server/src/simcore_service_webserver/exporter/formatters/formatter_v2.py

```diff
@@ -168,7 +168,7 @@ async def _write_sds_content(
     )
 
     # writing SDS structure with process pool to avoid blocking
-    with ProcessPoolExecutor(max_workers=1) as pool:
+    with non_blocking_process_pool_executor(max_workers=1) as pool:
```

Comment:
why not default executor?

Reply:
Composing XLS files is a CPU bound task, a ProcessPool is required

Reply:
understood with comment on the top.